IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcRadialExcavation/DocEntity.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcRadialExcavation" UniqueId="346a9b51-8391-440e-977a-daf2d8282779" DiagramNumber="1" BaseDefinition="IfcUndergroundExcavation" EntityFlags="32">
```

Comment:
Propose renaming to `IfcUndergroundRadialExcavation`.

Reply:
@larswik

Reply:
I think that we should go for IfcUndergroundFaceExcavation. Having a type doesn't work since face-/radial excavations are separate subtypes of underground excavation with their own sets of predefined types.